src/main/java/io/hydrox/inventorysummary/InventorySummaryPlugin.java

```diff
@@ -67,9 +70,10 @@
 	@Inject
 	private ItemManager itemManager;
 
-	@Getter
 	private List<String> whitelist = new ArrayList<>();
 
```

Comment:
It's personal preference more than anything, but for variables without annotations, I like to have them without extra lines between

Reply:
Sounds good. Fixed in latest commit.